adapters/mqtt-vertx-base/src/main/java/org/eclipse/hono/adapter/mqtt/Mapping.java

```diff
@@ -13,19 +13,21 @@
 
 package org.eclipse.hono.adapter.mqtt;
 
+import org.eclipse.hono.adapter.client.command.Command;
 import org.eclipse.hono.util.ExecutionContext;
 import org.eclipse.hono.util.RegistrationAssertion;
 import org.eclipse.hono.util.ResourceIdentifier;
 
 import io.vertx.core.Future;
+import io.vertx.core.buffer.Buffer;
 
 /**
  * A service for processing messages uploaded by devices before they are being
  * forwarded downstream.
  *
  * @param <T> The type of execution context supported by this mapping service.
  */
-public interface MessageMapping<T extends ExecutionContext> {
+public interface Mapping<T extends ExecutionContext> {
```

Comment:
I was struggling with this as well. `MessageMapping` as it was before didn't cover anymore what is was doing so that's why I renamed it. But I like your approach to rename the method and clarifying it that way.